Packs/Binalyze/Integrations/BinalyzeAIR/BinalyzeAIR_test.py

```diff
@@ -133,7 +134,7 @@ def test_air_isolate_command(requests_mock: Any) -> None:
     mock_response = util_load_json('test_data/test_isolate_success.json')
 
     client: Client = Client(
-        base_url=MOCK_URL,
```

Comment:
Let's add test for the get_profile_id function

Reply:
done